lib/migrations/migrate/migration-list-resolver.js

```diff
@@ -9,12 +9,14 @@ function listAll(migrationSource, loadExtensions) {
 // Lists all migrations that have been completed for the current db, as an
 // array.
 async function listCompleted(tableName, schemaName, trxOrKnex) {
-  await ensureTable(tableName, schemaName, trxOrKnex);
-
-  return await trxOrKnex
-    .from(getTableName(tableName, schemaName))
-    .orderBy('id')
-    .select('name');
+  // If the internal tables (migrations and locks) don't exist, then
+  // we haven't run any migrations at all.
+  return (await knexTablesExist(tableName, schemaName, trxOrKnex))
```

Comment:
I think this breaks migrations because of removing of ensureTable.
That not only check tables but create them with specific schema and insert a default lock row.
I think we still need to default creation tables